spec.bs

```diff
@@ -358,19 +666,67 @@ A <dfn export>fenced frame config</dfn> is a struct with the following [=struct/
 <div algorithm>
   The <dfn method for=Fence>reportEvent(<var ignore>event</var>)</dfn> method steps are:
 
-  1. <span class=XXX>Fill this out!</span>
+  1. Let *instance* be this [=traversable navigable=]'s [=fenced frame config instance=].
+
+  1. If *instance* is null, then return.
+
+  1. If *instance*'s [=fenced frame config instance/mapped url=] is null, then the behavior is
+     unspecified.
+
+  1. If this [=document=]'s [=origin=] and *instance*'s
+     [=fenced frame config instance/mapped url=]'s origin are not [=same origin=], then return.
+
+  1. If *instance*'s [=fenced frame config instance/fenced frame reporter=] is null, then return.
+
+  1. If *event* is a `DOMString`, run [=report a private aggregation event=] with *event* and
+     *instance*'s [=fenced frame config instance/fenced frame reporter=].
+
+  1. If *event* is a `FenceEvent`, run [=report an event=] with *event* and *instance*'s
```

Comment:
Same, you should be able to use {{FenceEvent}}, per https://speced.github.io/bikeshed/#:~:text=idl%2C%20covering%20the%20dfn%20autolinks%20for%20WebIDL%20types%20(the%20%7B%7Bfoo%7D%7D%20shorthand)